nuntium/models.py

```diff
@@ -108,7 +108,7 @@ class Message(models.Model):
     """Message: Class that contain the info for a model, \
     despite of the input and the output channels. Subject \
     and content are mandatory fields"""
-    author_name = models.CharField(max_length=512)
+    author_name = models.CharField(max_length=512, default='', blank=True)
```

Comment:
I think Django will want to generate a migration for this change, but it's not in this PR.